benchmarks/imagenet/vitb16/main.py

```diff
@@ -100,10 +108,11 @@ def main(
                 strategy=strategy,
             )
 
+        eval_metrics: EvalMetrics = []
```

Comment:
What do you think about aggregating all the metrics in a simple dictionary?

```suggestion
        eval_metrics = {}
```